app/helpers/spree/base_helper_decorator.rb

```diff
@@ -22,9 +20,21 @@ def supported_currencies_for_sale_price
     try(:supported_currencies) || [ _current_currency ]
   end
 
+  def gte_solidus_2_4?
+    gte_solidus?('2.4.0')
+  end
+
+  def gte_solidus_2_5?
+    gte_solidus?('2.5.0')
+  end
+
+  def gte_solidus?(version)
+    Spree.solidus_gem_version >= Gem::Version.new(version)
```

Comment:
I think you can use https://github.com/solidusio/solidus_support/blob/master/lib/solidus_support.rb#L7, right?